tools/interactive/interactive_pampa.xml

```diff
@@ -20,9 +20,9 @@
     ]]>
     </command>
     <inputs>
-        <param name="unitobs" type="data" multiple="true" format="xml" optional="true"  label="Input unitobs file"/>
-        <param name="obs" type="data" multiple="true" format="xml" optional="true"  label="Input observation file"/>
-        <param name="ref" type="data" multiple="true" format="xml" optional="true"  label="Input reference file"/>
+        <param name="unitobs" type="data" multiple="true" format="csv" optional="true"  label="Input unitobs file"/>
```

Comment:
Tabular is not possible?

Reply:
for now the app is using csv, BUT directly I was thinking to work on it so we can use tabular instead.... Something to see for a upcoming version